incubator/java-spring-boot2/image/project/Dockerfile

```diff
@@ -62,6 +65,10 @@ COPY --chown=java_user:java_group --from=compile ${DEPENDENCY}/BOOT-INF/classes
 
 USER java_user
 
+COPY --from=compile /tmp/scc/ /tmp/scc/
```

Comment:
COPY --chown=java_user:java_group --from=compile /tmp/scc /tmp/scc

Reply:
Updated the PR, Thanks @ebullient